platine-management-service/src/test/java/fr/insee/survey/datacollectionmanagement/metadata/service/impl/CampaignServiceImplTest.java

```diff
@@ -348,4 +349,84 @@ void testIsCampaignOngoingFalse_WhenCampaignHasNoPartitioning(){
         assertThat(campaignServiceImpl.isCampaignOngoing(camp)).isFalse();
 
     }
+
+    @Test
+    @DisplayName("Get Campaign Header with opended false if no partitioning is ongoing")
```

Comment:
suggestion: Get Campaign Header with status CLOSED if no partitioning is ongoing